src/current/_includes/v24.1/cdc/apache-pulsar-unsupported.md

```diff
@@ -0,0 +1,11 @@
+Changefeeds emitting to an Apache Pulsar sink do **not** support:
+
+- [`resolved`]({% link {{ page.version.version }}/create-changefeed.md %}#resolved-option)
```

Comment:
I have included all these options as unsupported, but I was able to start changefeeds with at least the top three into pulsar and receive messages with the respective timestamps (e.g., resolved). Given [Jay's issue](https://github.com/cockroachdb/cockroach/issues/118863), not sure if we want to include these or not. Maybe include until further testing?